.rubocop.yml

```diff
@@ -16,7 +16,7 @@ AllCops:
     - 'lib/deploy/*'
     - 'node_modules/**/*'
     - 'vendor/**/*'
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.7
```

Comment:
Huh, looks like you updated the target ruby version here. Not sure why CodeClimate is complaining...

Reply:
Turns out our version of Rubocop was too old, so I upgraded it